pages/common/jq.md

```diff
@@ -0,0 +1,23 @@
+# jq
+
+> A lightweight and flexible command-line JSON processor.
+
+- Output JSON file:
+
+`jq '' {{file}}`
+
+- Read JSON objects from file, into array, and output:
+
+`jq '' -s {{file}}`
```

Comment:
@maximpertsov sorry to nitpick, but I would prefer if the expansion of `-s` ("stream", I suppose) was included in the command description, per the [guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines) (3rd bullet point). Ideally we should include the word and an explanation for it in parenthesis, or vice-versa.